client/dvm/rpc/src/eth_pubsub.rs

```diff
@@ -4,8 +4,7 @@ pub use dvm_rpc_core::EthPubSubApiServer;
 use std::{collections::BTreeMap, iter, marker::PhantomData, sync::Arc};
 // --- crates.io ---
 use dvm_rpc_core::EthPubSubApi::{self as EthPubSubApiT};
-use dvm_rpc_runtime_api::EthereumRuntimeRPCApi;
-use ethereum::{BlockV0 as EthereumBlock, Log as EthereumLog, Receipt as EthereumReceiptV0};
+use ethereum::{BlockV2 as EthereumBlock, Log as EthereumLog, Receipt as EthereumReceiptV0};
```

Comment:
Still, using `EthereumReceiptV0`?

Reply:
Good catch. This is a known bug https://github.com/paritytech/frontier/pull/541. Just let it go, it will be fixed in #1090 .